install_config/install/docker_registry.adoc

```diff
@@ -1013,6 +1013,7 @@ storage:
     readonly:
       enabled: false
 ----
+<1> This entry is *mandatory* for image pruning to work properly.
```

Comment:
Do you think that the key naming `storage:delete:enabled` isn't sufficient enough?